cluster-autoscaler/cloudprovider/azure/azure_scale_set_test.go

```diff
@@ -65,6 +65,10 @@ func newTestVMSSList(cap int64, name, loc string, orchmode compute.Orchestration
 	return []compute.VirtualMachineScaleSet{
 		{
 			Name: to.StringPtr(name),
+			Tags: map[string]*string{
+				"aks-managed-poolName": to.StringPtr(name),
+			},
```

Comment:
We should move this entire structure to be more high fidelity, so going to live this change in even tho we don't need it anymore after I changed to parsing scaleset from provider id